setup.md

```diff
@@ -69,10 +69,13 @@ and the [CMIP6 Data Request](http://clipc-services.ceda.ac.uk/dreq/index.html).
 
 - A full list of [all ESGF nodes](https://esgf.llnl.gov/nodes.html).
 
+- A good [tutorial](https://esgf.github.io/esgf-user-support/user_guide.html#data-search-and-download)
```

Comment:
An alternative/additional suggestion could be to use the data search from climate4impact: https://climate4impact.eu/impactportal/data/esgfsearch.jsp

Reply:
@Peter9192 thanks. your suggestion is added to the list.